public/locales/en-US.json

```diff
@@ -148,9 +148,9 @@
     "header_subscriptions_cores": "Cores",
     "header_subscriptions_sockets": "Sockets",
     "header_Transfer-gibibytes": "Data transfer",
-    "measurement_Instance-hours": "{{total}} GiB hours",
-    "measurement_Storage-gibibytes": "{{total}} GiB",
-    "measurement_Transfer-gibibytes": "{{total}} GiB",
+    "measurement_Instance-hours": "{{total}} GB hours",
+    "measurement_Storage-gibibytes": "{{total}} GB",
```

Comment:
THIRD (missed this one by concentrating on my first 2 comments)
Change storage measurement here to GB hours, correct?

Reply:
So that might be a space problem... but yes that change would make it match the graph, @bclarhk ?
![Screen Shot 2022-02-23 at 1 06 21 PM](https://user-images.githubusercontent.com/3761375/155380205-a3206cbe-927e-4a5a-b5bc-356d33da66af.png)
![Screen Shot 2022-02-23 at 1 06 13 PM](https://user-images.githubusercontent.com/3761375/155380213-640d0bef-877f-4b6c-a652-bff4424b7ce4.png)

Reply:
@jlprevatt confirm this line stays the same?

Reply:
Storage: 
"GB hours"